### DISCUSSION TITLE
[Feature Request] Zoom Button Snap Panel.

### DISCUSSION THREAD

**Discussion by bigplayer-ai** [rxhanson/RectanglePro-Community#407] [Ideas]
[Feature Request] Zoom Button Snap Panel
Hi,

I would like to request a feature that allows the Snap Panel to appear below the zoom button, similar to how Moom does it. Please see the attached image for reference.

![Snap_Panel](https://github.com/rxhanson/RectanglePro-Community/assets/119108387/27de2e04-6ddf-4458-a227-e0692bdf7478)

Thank you,

**Comment by rxhanson** [comment]
Thanks for the idea, I'll add a backlog item to think this one through.

**Reply by bigplayer-ai** [reply]
For additional context, Moom application on the App Store does this.

Best Regards,

**Comment by vault21** [comment]
I literally came here to give the same feedback, so I was going to ask for this feature if I didn't find this thread. Right now this is the only missing feature from Rectangle that holds me back switching from Magnet and Moom. If Rectangle Pro adds this Zoom Button Panel, I'll just make the switch instantly

**Reply by rxhanson** [reply]
So that I better understand, are you simply looking for more customization than what comes out of the box in macOS?

**Reply by bigplayer-ai** [reply]
after macOS sequoia update I see this is redundant, because it is built in by Apple...
@vault21 are u using older OS?

**Reply by vault21** [reply]
I'm using Sequoia 15.4.1 (Latest for now), so I'm aware that MacOS natively supports window snipping now, but it's not customizable and it doesn't support custom dimensions. That's why I'm using third party apps for window management.

I'm currently using Moom just like the original poster mentioned above. Moom and Magnet allows you to make use of that green button (I think it's called zoom or maximise button), so when you hover your mouse cursor they show you their popup instead of MacOS popup. This still works on Sequoia. You can also modify and customise the window arrangement options you see on that popup menu. In my opinion, Moom does it better than Magnet. Magnet shows it as a list but Moom shows it as boxes similar to MacOS native feature and you can customize it nicely in the settings with a grid system.

Here are some screenshots:

**MacOS Native:**
<img width="366" alt="MacOS Native" src="https://github.com/user-attachments/assets/476388d8-7388-44ef-b52f-2851910b58bc" />

**Magnet:** "Resize Below Browser" preset you see here is a custom setting I generated.
![Magnet](https://github.com/user-attachments/assets/4c8966c3-4c51-4408-aa3c-5522c741b009)

**Moom:**
![Moom](https://github.com/user-attachments/assets/c32ad98e-a052-42f6-9461-34d7df50d8a8)

**Reply by rxhanson** [reply]
Thanks for the additional details. I'll update my backlog with this info and add your +1.
